.coafile

```diff
@@ -0,0 +1,16 @@
+[Default]
+
+[autopep8]
+bears = PEP8Bear, PycodestyleBear
+pycodestyle_ignore = E121, E123, E126, E226, E704,
+                     W503, W504
+
+default_actions = PEP8Bear: ApplyPatchAction
+
+[jinja2]
+bears = Jinja2Bear
+files = **.jinja2, **.jj2
```

Comment:
there are no .jj2 files in this repo

Reply:
I saw there were jj2 files on other repos and I thought that those kind of files could be included here in the future as well.